src/everest/gui/everest_client.py

```diff
@@ -91,9 +92,17 @@ def passthrough_ws_events() -> None:
 
         return event_queue, monitor_thread
 
-    def create_run_model_api(
-        self, queue_system: str, runpath_format_string: str
-    ) -> BaseRunModelAPI:
+    def create_run_model_api(self) -> BaseRunModelAPI:
```

Comment:
I don't understand why this needs to change in this way and if you want to refactor this I would argue that it should be moved to EverestMainWindow